overlord/ifacestate/ifacestate.go

```diff
@@ -198,6 +205,9 @@ func connect(st *state.State, plugSnap, plugName, slotSnap, slotName string, fla
 	if flags.ByGadget {
 		connectInterface.Set("by-gadget", true)
 	}
+	if flags.SkipSetupProfiles {
+		connectInterface.Set("skip-setup-profiles", true)
```

Comment:
s/skip/delay/ ? it's not really skipping the action, rather delaying it until later time

Reply:
Good idea, done.